cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -137,6 +140,15 @@ class SubstraitToVeloxPlanConverter {
     planNodeId_ = planNodeId;
   }
 
+  void setSplitInfos(std::vector<std::shared_ptr<SplitInfo>> splitInfos) {
+    splitInfoIdx_ = 0;
```

Comment:
> int32_t splitInfoIdx_{0};

Why do we need to set split info index as 0? Its initial value should already be 0.

Reply:
make sure each call to `setSplitInfos` will reset `splitInfoIdx_` correctly.